_projects/git_task_list.md

```diff
@@ -0,0 +1,115 @@
+---
+collaborating_projects:
+  - gsoc-prep-tasks
+desc: "List open issues and pull request from various issue tracker of a specified organization."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - https://github.com/coala/coala/issues/4080
+markdown: git_task_list.md
+mentors:
+  - blazeu
+  - yukiisbored
+  - yashovardhanagrawal
+name: "Git Task List"
+requirements:
+  - "The applicant should have at least one patch merged to any of the coala repositories."
+  - "Familiarity with Git, GitHub API, Phabricator API, and GitLab API"
+  - "Familiarity with EmberJS and GitHub Pages"
+status: in_progress
+tags:
+  - community
+  - web
+---
+The basic idea is an extended version of [GSOC Task List](http://github.com/summerofcode/gsoc-prep-tasks).
+
+In this project, tasks means open issues and unmerged pull request coming from
+GSOC open source projects.
+
+[GSOC prep task](http://summerofcode.github.io/gsoc-prep-tasks) is an issue
+finding tools that is (currently) integrated only with GitHub. It helps filter
+out the GitHub issues and only show the issues that are newcomer friendly and
+coming from limited GSoC organization. It limits the organization because
+different organization use different kind of issue tracking such as Phabricator
+and GitLab.
+
+**Git Task List** is conceptually the same as [GSOC prep task](http://summerofcode.github.io/gsoc-prep-tasks).
+Git Task List target a broader task type which include pull request review as
+the task. The goal of this project is to help developer to keep contributing to
+open source project by providing a sets of tasks that has been filtered using
+predefined keywords.
+
+In coala, this project will help sharing the tasks of reviewing a pull request
+on a project accross GitHub and GitLab. It will help fix the problem where
+developer spend too much time finding PR that haven’t been reviewed. This
+projects aims to getting more developer doing reviews works and sharing reviews
+workload, by providing an easy way to find issues that has not been reviewed
+both on http://github.com/coala and http://gitlab.com/coala.
+
+This will also help engage the developer to keep contributing to a project where
+they can’t find any issue that they can fix or keep the newcomer issue for new
+comer as learning path and developer with more experience can contribute by
+reviewing newcomer’s work PR.
+
+The goal of this projects is a static-site javascript web page where the user
```

Comment:
I think `static-site javascript web page` is really verbose, `static site` is enough.